contents/docs/libraries/ios/_snippets/autocapture.mdx

```diff
@@ -9,7 +9,7 @@ PostHog autocapture automatically tracks the following events for you:
 -   **$screen** - when the user navigates (if using `UIViewController`)
 -   **$autocapture** - when the user interacts with elements in a screen (if using `UIKit`)
 
-> 🚧 **Note:** `$autocapture` is currently supported only in UIKit.
+> 🚧 **Note:** `$autocapture` captures interactions with UIKit elements. Some SwiftUI views use UIKit under the hood (e.g., `TextField` → `UITextField`, `Toggle` → `UISwitch`) and will also be autocaptured.
```

Comment:
@ioannisj 
https://posthog.com/docs/libraries/ios/configuration#autocapture-configuration

> (UIKit only) Whether the posthog client should automatically make a capture call when the user interacts with an element in a screen.

i think we should make that sentence clearer as well